agents/go-agents/exec-agent/main.go

```diff
@@ -105,14 +119,15 @@ func main() {
 	log.Fatal(server.ListenAndServe())
 }
 
-func getHandler(h http.Handler) http.Handler {
-	// required authentication for all the requests, if it is configured
-	if config.authEnabled {
-		cache := auth.NewCache(time.Minute*time.Duration(config.tokensExpirationTimeoutInMinutes), time.Minute*5)
-		return auth.NewCachingHandler(h, config.apiEndpoint, droppingRPCChannelsUnauthorizedHandler, cache)
+func wrapWithAuth(h http.Handler, mapping string) http.Handler {
```

Comment:
I expect that `mapping` value will say which should be wrapped, but actually, it is `noAuthPattern`. Please consider to renaming this variable.

Reply:
Go default library doesn't have negative lookahead syntax in regexp. Because of that, I don't know how to do that without complicating the code and significantly rewriting it. Feel free to do a suggestion if you have any idea.

Reply:
`ignoreMapping`?

Reply:
+1